src/styles/blocks/card.scss

```
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-color;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $light-color;
+    margin-bottom: 16px;
+  }
+
+  &__rewiew {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__rewiew-count {
+    color: $dark-color;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    background: $light-color;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 18px;
+    background: $dark-color;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    font-weight: 700;
    font-size: 12px;
    line-height: 18px;
    background: $dark-color;
    
    &-label {
     // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```